src/services/parser/__test__/sqs-incoming-message-parser.test.js

```diff
@@ -1,13 +1,17 @@
+import { readFileSync } from "fs";
+import * as path from "path";
 import { parse } from '../sqs-incoming-message-parser';
 import expect from 'expect';
 import { XmlParser } from '../xml-parser/xml-parser';
 
+
```

Comment:
Could we please get rid of this whitespace? There's a couple of bits in this file